metadata/latest/Azure.AI.FormRecognizer.json

```diff
@@ -17,5 +17,7 @@
     "Azure.AI.FormRecognizer.Models",
     "Azure.AI.FormRecognizer.Training",
     "Microsoft.Extensions.Azure"
-  ]
+  ],
+  "DocsCiConfigProperties": "tfm=netstandard2.0",
+  "MSDocService": "placeholder"
```

Comment:
Without this value, the package will be filtered out by the 'New' to `true` filter.